pages/pipelines/managing_log_output.md

```diff
@@ -209,4 +209,10 @@ To set up a private build log archive storage:
     }
     ```
 
+    >📘
+    > Folder structure and file format will be as follow: {BUILDKITE_PIPELINE_ID}/{BUILDKITE_BUILD_ID}/{BUILDKITE_JOB_ID}.log. This file format and folder structure is currently not customisable. 
+    
+    > 🚧 Log Retention
+    > After build logs are archived to customer's S3 buckets, Buildkite won't retain a copy of those logs internally. 
```

Comment:
I think this belongs in the intro paragraph rather than a callout. I've added a suggestion to move include it there.

```suggestion
```